### PR TITLE
feat: add project list section in home page

### PR DESCRIPTION
**Pull request by divyanshugarg36**
Add a project list on the home page showing 2 projects in the beginning

### CODE REVIEW

File: src/components/ProjectSection.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  const projects = [
+    {
+      title: t('ABOUT.ABOUT_1.TITLE'),
+      description: t('ABOUT.ABOUT_1.DESCRIPTION'),
+    },
+    {
+      title: t('ABOUT.ABOUT_2.TITLE'),
+      description: t('ABOUT.ABOUT_2.DESCRIPTION'),
+    },
+  ];
```
Comment (theamanjs) [inline src/components/ProjectSection.tsx]: Can this be moved to another constant file?

File: src/components/common/ProjectCard.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+}
+
+export const ProjectCard: React.FC<IProps> = ({
+  className, title, description, isImage,
+}) => (
+  <div className={classNames('project-card', className)}>
+    <div className="project-card-body">
+      <h1>{title}</h1>
+      <p>{description}</p>
+      <a>Visit Github</a>
```
Comment (theamanjs) [inline src/components/common/ProjectCard.tsx]: Should this string come from i18n?

**Review by theamanjs — CHANGES_REQUESTED** [review]

File: src/components/ProjectSection.tsx (hunk 1 above)
Comment (divyanshugarg36) [reply src/components/ProjectSection.tsx]: This has been fixed

File: src/components/common/ProjectCard.tsx (hunk 2 above)
Comment (divyanshugarg36) [reply src/components/common/ProjectCard.tsx]: Moved to i18n

Comment (theamanjs) [reply src/components/common/ProjectCard.tsx]: coool

**Review by theamanjs — APPROVED** [review]

**Pull request merged**